quint/src/ir/quintTypes.ts

```diff
@@ -144,6 +144,19 @@ export type EmptyRow = { kind: 'empty' }
 
 export type Row = ConcreteFixedRow | ConcreteRow | VarRow | EmptyRow
 
+/*
+ * Gives all of a row's field names
+ */
+export function rowFieldNames(r: Row): string[] {
```

Comment:
This was a helper I couldn't end up using in this go, but is generally useful IMO.